immunisations/README.md

```diff
@@ -0,0 +1,149 @@
+             },
+             {
+                 "text": "End Date: 2010-01-18T11:41:00+00:00"
+             }
+         ]
+     }
+ },
+```
+</details>
+
+
+## XML HL7 > JSON FHIR
+
+An Immunization is primarily mapped from an Observation Statement.
+
+| Mapped to (XML HL7)                                | Mapped from (JSON FHIR / other source )              |
+|----------------------------------------------------|------------------------------------------------------|
+| id / root                                          | `id`                                                 |
+| code / display                                     | `extension.valueExtension.valueCodeableConcept.text` |
+| Participant / typeCode / agentRef / id /root       | `practitioner.actor.reference`                       |
+| statusCode                                         | `status`                                             |
+| effectiveTime                                      | `date`                                               |
+| availableTime                                      | `date`                                               |
+| pertiinentInformation / pertiantAnnotation / text  | `note.text`                                          |
+| Participant / typecode / agentRef / id / root      | `practitioner.actor.reference`                       |
+
+
+<details><summary>Example XML</summary>
+
+```XML
+<ObservationStatement classCode="OBS" moodCode="EVN" xmlns="urn:hl7-org:v3">
+    <id root="immunization-id"/>
+    <code code="33879002" codeSystem="2.16.840.1.113883.2.1.3.2.3.15"
+          displayName="Haemophilus influenzae type B and meningitis C vaccination">
+    </code>
+    <Participant typeCode="PRF" contextControlCode="OP">
+        <agentRef classCode="AGNT">
+            <id root="9C1610C2-5E48-4ED5-882B-5A4A172AFA35"/>
+        </agentRef>
+    </Participant>
+    <statusCode code="COMPLETE"/>
+    <effectiveTime>
+        <high value="20100118114100"/>
+    </effectiveTime>
+    <availabilityTime value="20100118114100"/>
+    <pertinentInformation typeCode="PERT">
+        <sequenceNumber value="+1"/>
+        <pertinentAnnotation classCode="OBS" moodCode="EVN">
+            <text>Primary Source: true Location: EMIS Test Practice Location Manufacturer:
+                another company Batch: past2003 Expiration: 2003-01-17 Site: Right arm GMS : Not
+                GMS
+            </text>
+        </pertinentAnnotation>
+    </pertinentInformation>
+    <Participant typeCode="PRF" contextControlCode="OP">
+        <agentRef classCode="AGNT">
+            <id root="9A5D5A78-1F63-434C-9637-1D7E7843341B"/>
+        </agentRef>
+    </Participant>
+</ObservationStatement>
+```
+
+</details>
+
+## Further documentation
+
+[Immunistion History - FIHR API](https://digital.nhs.uk/developer/api-catalogue/immunisation-history-fhir)

```

Comment:
I would like to `https://fhir.nhs.uk/STU3/StructureDefinition/CareConnect-GPC-Immunization-1`